### PR TITLE
Zemanta adapter: rename adapter to Outbrain

### PR DESCRIPTION
**Pull request by rokostik**
Hey, this renames the Zemanta adapter to Outbrain.
When we first made the decision to expose the Zemanta adapter as Outbrain we only supported prebid.js that this with minimal effort with the alias function. But now we also have a server adapter and noticed that it could cause confusion since aliases work a bit differently there so we decided to fully rename both adapters to Outbrain.

We currently have no server integrations so this doesn't break anything.

Link to the PR on the docs repo at prebid/prebid.github.io#2847

### CODE REVIEW

**Comment by SyntaxNode** [comment]
Considering the Zemanta adapter was just released yesterday, I don't see harm in renaming it. Thoughts @hhhjort @mansinahar ?

**Comment by hhhjort** [comment]
I am fine with this since it was just released. I will also note improved alias support is on the roadmap, but can always alias Zemanta to Outbrain.

**Review by hhhjort — APPROVED** [review]
LGTM

File: config/config.go [hunk 1]
```diff
@@ -622,6 +622,7 @@ func (cfg *Configuration) setDerivedDefaults() {
 	setDefaultUsersync(cfg.Adapters, openrtb_ext.BidderNoBid, "https://ads.servenobid.com/getsync?tek=pbs&ver=1&gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redirect="+url.QueryEscape(externalURL)+"%2Fsetuid%3Fbidder%3Dnobid%26uid%3D%24UID")
 	setDefaultUsersync(cfg.Adapters, openrtb_ext.BidderOpenx, "https://rtb.openx.net/sync/prebid?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&r="+url.QueryEscape(externalURL)+"%2Fsetuid%3Fbidder%3Dopenx%26gdpr%3D{{.GDPR}}%26gdpr_consent%3D{{.GDPRConsent}}%26uid%3D%24%7BUID%7D")
 	setDefaultUsersync(cfg.Adapters, openrtb_ext.BidderOneTag, "https://onetag-sys.com/usync/?redir="+url.QueryEscape(externalURL)+"%2Fsetuid%3Fbidder%3Donetag%26gdpr%3D{{.GDPR}}%26gdpr_consent%3D{{.GDPRConsent}}%26uid%3D%24%7BUSER_TOKEN%7D")
+	setDefaultUsersync(cfg.Adapters, openrtb_ext.BidderOutbrain, "https://prebidtest.zemanta.com/usersync/prebidtest?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&cb="+url.QueryEscape(externalURL)+"%2Fsetuid%3Fbidder%3Doutbrain%26uid%3D__ZUID__")
```
Comment (SyntaxNode) [inline config/config.go:625]: Should the URL still point to zemanta.com?

Comment (rokostik) [reply config/config.go:625]: Yes, the URLs stay the same.

**Review by SyntaxNode — APPROVED** [review]

**Pull request merged**